rhel7/profiles/ospp.profile

```diff
@@ -365,7 +365,7 @@ selections:
     - var_accounts_tmout=10_min
     - var_password_pam_difok=8
     - var_password_pam_maxclassrepeat=4
-    - var_password_pam_maxrepeat=2
+    - var_password_pam_maxrepeat=3
```

Comment:
DISA STIG profile requires `maxrepeat=3`, but I'm not sure about OSPP Policy.
Also, default selector for `var_password_pam_maxrepeat` is 3 already. If OSPP requires it to be 3, this line can be removed.